LICENSE

```diff
@@ -249,60 +252,63 @@ Libraries
 - OpenTelemetry.Instrumentation.Wcf
 are under the following copyright:
 Copyright The OpenTelemetry Authors under Apache License Version 2.0
-(https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/LICENSE).
+(<https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/LICENSE>).
 
 Library OpenTracing is under the following copyright:
 Copyright The OpenTracing Authors under Apache License Version 2.0
-(https://github.com/opentracing/opentracing-csharp/blob/master/LICENSE).
+(<https://github.com/opentracing/opentracing-csharp/blob/master/LICENSE>).
 
 Libraries
-- System.Diagnostics.DiagnosticSource,
-- System.Security.Permissions,
+
+- Microsoft.Extensions.Configuration,
 - Microsoft.Extensions.Configuration.Abstractions,
 - Microsoft.Extensions.Configuration.Binder,
-- Microsoft.Extensions.Configuration.EnvironmentVariables,
```

Comment:
There are only two non-trivial changes in this file. This is No. 2 of the two.

2. Removed `Microsoft.Extensions.Configuration.EnvironmentVariables`.